lib/linguist/languages.yml

```diff
@@ -1659,13 +1659,16 @@ HTML+Django:
   tm_scope: text.html.django
   group: HTML
   extensions:
-  - ".mustache"
   - ".jinja"
+  - ".mustache"
```

Comment:
Well, that's true, isn't it? Mustache is primarily a js templating language. And the django just says "django templating language" (no mention of Mustache at all), and calls out jinja2 as an alternative. 

https://docs.djangoproject.com/en/1.10/topics/templates/